modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/Sidebar.js

```diff
@@ -71,17 +71,19 @@ const Sidebar = ({children, onClose = noop, open = true}) => {
 		}
 	}, [delay, open]);
 
+	useEffect(() => {
+		const body = document.querySelector('body');
```

Comment:
Why not just `document.body`?

Reply:
copied from an example in codepen using `.body', just forgot to change it completely 😅